src/common/valtext.cpp

```diff
@@ -332,6 +312,56 @@ void wxTextValidator::OnChar(wxKeyEvent& event)
     event.Skip(false);
 }
 
+bool wxTextValidator::IsValid(const wxUniChar& c) const
+{
+    if ( !m_validatorStyle ) // no filtering if HasFlag(wxFILTER_NONE)
+        return true;
+
+    if ( HasFlag(wxFILTER_SPACE) && wxIsspace(c) )
```

Comment:
Why handle this specially? It seems like it could be handled with the other filters below.